### PR TITLE
Removed policy with status objects

### PR DESCRIPTION
**Pull request by compae**

### CODE REVIEW

File: web/src/templates/modal/policy-info-modal.tpl.html [hunk 1]
```diff
@@ -19,8 +19,7 @@
         <p>{{vm.statusInfo}}</p>
       </div>
 
-      <div data-ng-if="submissionId"
```
Comment (EvaMGC) [inline web/src/templates/modal/policy-info-modal.tpl.html]: Change to data-ng-if="vm.submissionId"

File: web/src/views/policies.html [hunk 2]
```diff
@@ -22,26 +22,33 @@
         <thead class="c-table__thead">
         <tr>
           <th class="c-table__td c-col--2">{{"_NAME_" | translate}}</th>
-          <th class="c-table__td c-col--2">{{"_DESCRIPTION_" | translate}}</th>
```
Comment (EvaMGC) [inline web/src/views/policies.html]: <th class="c-table__td c-col--2">{{"_NAME_" | translate}}</th>
          <th class="c-table__td c-col--2 c-table--centered-content">{{"_STATUS_" | translate}}</th>
          <th class="c-table__td c-col--3">{{"_STATUS_INFORMATION_" | translate}}</th>
          <th class="c-table__td c-col--2">{{"_SUBMISSION_ID_" | translate}}</th>
          <th class="c-table__td c-col--2 c-table--centered-content">{{"_DEPLOY_MODE_" | translate}}</th>
          <th class="c-table__td c-col--1"></th>
          <th class="c-table__td c-col--1"></th>

File: web/src/views/policies.html [hunk 3]
```diff
@@ -22,26 +22,33 @@
...
-          <td class="c-table__td c-table__td--no-wrap c-table__td--ellipsis">{{policyData.description}}</td>
           <td class="c-table__td c-col--1 c-table--centered-content">{{policyData.status}}</td>
           <td class="c-table__td c-table__td--no-wrap c-table__td--ellipsis">{{policyData.statusInfo}}</td>
           <td class="c-table__td c-table__td--no-wrap c-table__td--ellipsis">{{policyData.submissionId}}</td>
           <td class="c-table__td c-table__td--no-wrap c-table__td--ellipsis c-table--centered-content">{{policyData.lastExecutionMode}}</td>
 
           <td class="c-table__td c-table__td--to-right c-col--1 c-col--to-right c-row--to-right">
+            <c-icon-link icon-class="'c-table__icon c-icon c-icon--is-clickable c-icon--circle-button c-icon--call-to-action icon-help1'"
+                         data-ng-if="policyData.clusterUI"
+                         data-ng-click="policyData.clusterUI"
```
Comment (EvaMGC) [inline web/src/views/policies.html]: Remove this line

**Comment by ahvargas** [comment]
LGTM , tested in spark 1.5

**Pull request merged**